hathor/builder.py

```diff
@@ -362,7 +367,7 @@ def create_resources(self, args: Namespace) -> server.Site:
         )
         from hathor.websocket import HathorAdminWebsocketFactory, WebsocketStatsResource
 
-        settings = HathorSettings()
+        settings = DefaultHathorSettings()
```

Comment:
Why are you mixing different settings? Shouldn't all created objects use the same?

Reply:
I have rolled back that change as I found a better way to do it and it's not necessary anymore.

Reply:
Ok!